.storybook/preview.js

```diff
@@ -9,13 +10,66 @@ import 'focus-visible';
 import '../src/index.scss';
 import './preview.scss';
 
+/**
+ * @typedef { [String, import('@storybook/client-api/dist/types').StoreItem] } StoryItem
+ */
+
 // Accessibility testing via aXe
 addDecorator(withA11y);
 
 // Theme selection from stories
 const themes = [{ name: 'Dark', class: 't-dark', color: colors.primaryBrand }];
 addParameters({ themes });
 
+/**
+ * Get the category from a `storySort` story
+ * @param {StoryItem} story - Item passed to `addParameters.options.storySort`
+ * @returns {string} - Story's category, the first part of the `StoreItem.kind`
+ * value, which is a string delimited with forward slashes
+ */
+const getStoryCategory = (story) => story[1].kind.split('/')[0];
+
+/**
+ * Define an ordered list of story categories. These should match the first part
+ * of the `<Meta>` component's `title` attribute found in each story (`.mdx` file)
+ */
+const orderedCategories = [
+  'Getting Started',
+  'Design',
+  'Design Tokens',
+  'Objects',
+  'Components',
+  'Utilities',
+  'Themes',
+  'Third-Party',
+  'Prototypes',
+];
+/**
+ * Sanitize (slufigy) each entry in the list of categories, to normalize odd characters,
+ * capitalization, and use of spaces vs dashes. Do this outside of the `storySort`
+ * sorting function
+ */
```

Comment:
This comment focuses on _what_ you're doing, but doesn't include _why_. What do odd characters, capitalization, spaces, etc. impact in terms of sorting?

Reply:
I've updated the comment here and added one inside the `storySort` function, in an attempt to explain _why_ I did this. Whether this is "needed" is unclear to me. I just felt a little uneasy about having hard-coded strings that might deviate from the strings in `Meta` `title` attributes in simple ways, but then again, those are also just entered by hand. Makes me wish these were all tokenized, somehow.

If you feel this is excessive/unnecessary, I'll just pull it out and simplify.

Reply:
> Sanitize (slufigy) each entry in the list of categories, to normalize odd characters, capitalization, and use of spaces vs dashes.

You've added more detail about _what_ you are doing, but that wasn't the issue. The issue is _why_ you are doing it. _Why_ are we normalizing these things? What happens if we don't?

I think you're strategy of slug-ifying is fine. What I can't tell is whether or not it's even necessary, because I don't know what prompted it. Was the sorting wrong beforehand? Are there certain characters it is incompatible with?

If the answer is "I just had a feeling this _might_ be necessary someday," that probably doesn't justify the inclusion of this logic here. But if you had a non-hypothetical reason, like "spaces weren't sorted the way we would expect them to be" or something like that, it's critical we include that detail in our comments.

Reply:
> If the answer is "I just had a feeling this might be necessary someday," that probably doesn't justify the inclusion of this logic here

This is the case. I'm going to remove the sanitization and streamline. If needed in the future, we can re-include.